src/LaborMarket/LaborMarket.sol

```diff
@@ -57,12 +58,11 @@ contract LaborMarket is
                 msg.sender,
                 configuration.delegateTokenId
             ) >= 1) ||
-                (participationBadge.balanceOf(
-                    msg.sender,
-                    configuration.participationTokenId
-                ) >=
-                    (network.baseProviderThreshold() *
-                        configuration.repParticipantMultiplier)),
+                (reputationModule.getAvailableReputation(
```

Comment:
Please minify the use of copied code at lines 77 - 80 into a single function that can be reused. At all times, smart contracts should follow [DRY](https://www.digitalocean.com/community/tutorials/what-is-dry-development).

Reply:
Will do